openfecli/parameters/misc.py

```diff
@@ -22,3 +22,10 @@
     default=False,
     help="Overwrite any partial charges present in the input molecules."
 )
+
+ADAPTIVE_SETTINGS = Option(
```

Comment:
I'm not sure we should be offering this as a flag, i.e. the CLI hand holds such that we should just offer the best settings at any given time(?)